services/usb-livesync-service-base.ts

```diff
@@ -39,7 +39,7 @@ class SyncBatch {
 					});
 				}
 				this.timer = null;
-			}, 500);
+			}, 250);
```

Comment:
And there's a comment there, I'm sure I'll wonder why we've chosen 250 next time I check this code. Also the explanation there doesn't give exact hints why 250 is the magical number, but I'll be happy in case we have the same comment :)